pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -34,7 +34,59 @@
 
 class EarlyStopping(Callback):
     r"""
-    Monitor a metric and stop training when it stops improving.
+    Monitor a metric and stop training when a certain validation metric stops improving.
+
+    .. code-block:: python
+
+        # 1. Import EarlyStopping callback
```

Comment:
The docs are nice but I don't think the api docstrings are the best place for these tutorials and user guides. I would avoid it.

Reply:
where would you suggest then? Is there a reason not to put in the docstring?